ProteoformSuiteGUI/LoadResults.cs

```diff
@@ -319,6 +315,12 @@ private void refresh_dgvs()
             {
                 dgv.Refresh();
             }
+
+            if (MdiParent != null) //doesn't work first time
```

Comment:
Doesn't work the first time?

Reply:
No I think it's when it's building before we've set the MdiParents